packages/waku/src/router/client.ts

```diff
@@ -928,6 +935,7 @@ const InnerRouter = ({
         prefetchRoute,
         routeChangeEvents,
         fetchingSlices: useRef(new Set<SliceId>()).current,
+        basePath: import.meta.env.WAKU_CONFIG_BASE_PATH
```

Comment:
`import.meta.env.WAKU_CONFIG_BASE_PATH` is a constant, so passing it around as prop seems odd. What about defining this constant at the top of this file?

```js
const BASE_PATH = import.meta.env.WAKU_CONFIG_BASE_PATH;
```